python/cuspatial/cuspatial/__init__.py

```diff
@@ -7,12 +7,7 @@
 )
 from .core.interpolate import CubicSpline
 from .core.spatial_window import points_in_spatial_window
-from .core.trajectory import (
-    derive,
-    distance_and_speed,
-    spatial_bounds,
-    subset_trajectory_id,
-)
+from .core.trajectory import derive, distance_and_speed, spatial_bounds
```

Comment:
Do we want to be re-exporting these functions to the top-level `cuspatial` namespace? Worried things like this could lead to confusion:
```
# derive what? should this be renamed?
cuspatial.derive(...)

# or should we force this?
cuspatial.trajectory.derive(...)
```

Reply:
I defer to @thomcom @kkraus14 .

Reply:
I'm not familiar with this API space as well where I would defer to @thomcom as well.

Reply:
IMO force `cuspatial.trajectory.derive`, because we can always add the other later, whereas removing one is way harder. 2cents.

Reply:
@trxcllnt proposed calling the function `device_trajectory`, which sounds good to me.

Reply:
Yeah having different names from C++ seemed unclear, so I changed Python to be the same:
```python
cuspatial.derive_trajectories()
cuspatial.trajectory_bounding_boxes()
cuspatial.trajectory_distances_and_speeds()
```